static/app/views/alerts/rules/metric/triggers/form.tsx

```diff
@@ -157,6 +157,11 @@ class TriggerFormContainer extends Component<TriggerFormContainerProps> {
   };
 
   getThresholdUnits(aggregate: string, comparisonType: AlertRuleComparisonType) {
+    // cls is a number not a measurement of time
+    if (aggregate.includes('measurements.cls')) {
```

Comment:
@taylangocmen Using the `measurementType()` function might be better as we add more non-duration based measurements to alerts in the future (e.g. Mobile Vitals): https://github.com/getsentry/sentry/blob/e0fe30d9831f77ed60727b3d168f69a59726f790/static/app/utils/discover/fields.tsx#L793-L798



```suggestion
    if (measurementType(aggregate) !== 'duration') {
```

Reply:
Just `measurementType(aggregate)` would ignore the non-vital type aggregates like `transaction.duration`, so a combination of the two is needed

Reply:
@taylangocmen Ah. In that case, `aggregateFunctionOutputType()` might be an option.

Reply:
Actually `measurementType(aggregate)` wouldn't work on alert `MEASUREMENTS` `fields` because the alert aggregate includes the aggregate function ie. `percentile(measurements.lcp,0.5)'` or `p99(measurements.lcp)` vs just `measurements.lcp` substring matching on every aggregate here isn't worth it.